### PR TITLE
Completed css-canvas-medical challenge

### PR DESCRIPTION
**Pull request by jeff-cruz**
Kapture 2022-03-19 at 20 52 24

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Jeffrey! I see a lot of usage of the `row` class and a couple of usages of the `column-half` class, but your `row` class doesn't have `display: flex` applied to it, and you applied `display: flex` to a lot of places individually. There would also be more column classes necessary for this layout than just `column-half` if layout classes were being used correctly. Try to go through this and revise your HTML and CSS to make better use of layout classes in their proper format described in the `css-layout-classes` exercise.

**Comment by jeff-cruz** [comment]
Kapture 2022-03-22 at 11 29 41

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    <!--CLINICIANS PREFER CANVAS-->
+    <div class="row clinician-container grey-background">
+      <div class="column-full">
+        <h1>Clinicians Prefer Canvas</h1>
+        <p>Physician-owned primary care practices across the country have <br>
+          already made Canvas the fabric of their care.
+        </p>
+      </div>
+
+      <div class="row">
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:62]: This is a `row` directly inside a `row`, you can probably just remove this element and make these two `column-half` elements into siblings of the above `column-full`

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+.container {
+  flex-basis: 75%;
+  margin: auto;
+}
+
+.row {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:13]: Don't pollute your layout classes, you should move `width` and `flex-wrap` to different classes you can add in addition to the `row` class

File: css-canvas-medical/styles.css [hunk 3]
```diff
@@ -0,0 +1,250 @@
+body {
+  background-color: white;
+}
+
+.container {
+  flex-basis: 75%;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:6]: `flex-basis` only works on children of an element set to `display: flex`. You should change this to `max-width`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Great work!! 🎉

**Pull request merged**